event-streams/src/main/java/com/redhat/cloud/platform/playbook_dispatcher/types/RunHostEvent.java

```diff
@@ -0,0 +1,181 @@
+package com.redhat.cloud.platform.playbook_dispatcher.types;
+
+import java.util.HashMap;
+import java.util.Map;
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "event_type",
+        "hostPayload"
+})
+public class RunHostEvent {
```

Comment:
@SteveHNH did you run the generator manually to create these classes? Would be great to add schema to plugin configuration in https://github.com/RedHatInsights/playbook-dispatcher/blob/master/event-streams/pom.xml#L151 so that the classes are automatically generated during maven build

Reply:
Done.